### ISSUE TITLE
[ISSUE] - cata classic - exception after login

### ISSUE THREAD

**Issue by ZeChrales** [Nevcairiel/Routes#19]
### Description

Exception just after login a char

### Routes Version

1.8.1

### World of Warcraft Version

Classic Wrath of the Lich King

### Testing Isolation

- [ ] No
- [ ] Yes, without any related optional dependencies.
- [ ] Yes, with related optional dependencies.

### Lua Error(s)

```true
Stack: Interface/AddOns/Routes/Modules/AutoShow.lua:25: table index is nil
[string "@Interface/AddOns/Routes/Modules/AutoShow.lua"]:25: in main chunk
```

### Reproduction Steps

1 login

**Comment by yssaril** [comment]
in the Routes/Modules/AutoShow.lua

change line 25 from this
`profession_to_skill[GetSpellInfo(170691)] = "Herbalism"`
to
`
	profession_to_skill[GetSpellInfo(170691) or GetSpellInfo(13614)] = "Herbalism"`

**Comment by Nevcairiel** [comment]
This is already fixed in 7ba55fd442c4ffe15defe7ae3c367c0085832468
